persistence-modules/spring-data-jpa-query-3/src/test/java/com/baeldung/spring/data/jpa/paging/CustomerServiceUnitTest.java

```diff
@@ -77,4 +77,14 @@ void givenAListOfCustomers_whenGetCustomers_thenReturnsDesiredDataAlongWithPagin
         assertEquals(expectedTotalElements, customers.getTotalElements());
         assertEquals(expectedTotalPages, customers.getTotalPages());
     }
+
+    void givenAPageOfCustomers_whenGetCustomerList_thenReturnsDesiredListData() {
```

Comment:
We are missing the @Test annotation otherwise the test is not executed.

Reply:
Oops, done